pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1408,6 +1408,9 @@ type WorkflowStep struct {
 	// Hooks holds the lifecycle hook which is invoked at lifecycle of
 	// step, irrespective of the success, failure, or error status of the primary step
 	Hooks LifecycleHooks `json:"hooks,omitempty" protobuf:"bytes,12,opt,name=hooks"`
+
+	// Priority is used if controller is configured to process limited number of steps in parallel. Steps with higher priority are processed first.
```

Comment:
I think this could be a simpler comment. E.g. "Priority controlles which step is scheduled first. Lower values are processed first."